### ISSUE TITLE
Assignment 10.1

### ISSUE THREAD

**Issue by Yahsflame** [theironyard-frontend-nashville/assignments#86]
https://medium.com/@bensigal/never-one-to-back-down-from-a-challenge-especially-involving-literature-and-creative-writing-i-92e4d1506e00

**Comment by egdelwonk** [comment]
I appreciate your literary prowess in your submission!

Can you also annotate each snippet with what you think it's doing and why you find it useful?

Also, there are a few items that need to be corrected:

```
var sup = function(greeting){return greeting + ',' + this.name};
    sup = _.bind(sup, {name: 'Will'}, 'Sup');
    console.log(sup()); // logs 'Sup,Will'
```
- `// yay returns the objects in order with 5 added to each of the number values`
  The order is not guaranteed. Can you rephrase?
- Using _.mixin creates a new method on the underscore object.
  `//this will produce a function equal to "Oh My GAH, Will!"`, this should say it  will produce a new function that when invoked will return the output of "Oh My GAH, Will!"
- Can you elaborate on what iteratee is doing?

**Comment by Yahsflame** [comment]
https://medium.com/@bensigal/never-one-to-back-down-from-a-challenge-especially-involving-literature-and-creative-writing-i-92e4d1506e00 

here you go!

**Comment by Yahsflame** [comment]
https://medium.com/@bensigal/never-one-to-back-down-from-a-challenge-especially-involving-literature-and-creative-writing-i-92e4d1506e00

completed with the changes we discussed.
